docs/resources/azurerm_mysql_server.md.erb

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
This doc appears to have suffered a small formatting issue! It seems to be missing its header section e.g.
```

---
title: About the azurerm_mysql_server Resource
platform: azure
---

.... etc
```